### ISSUE TITLE
Revamping the UI

### ISSUE THREAD

**Issue by CH1NRU5T** [amanmalakar007/Scientific-Calculator#3]
The calculator looks good but is not in centre, the UI is a little messed up.

Should I try to improve the UI?

**Comment by the-cyberman** [comment]
Yeah sure. Looking forward to see a version 2.0 of this old fashioned
calculator. Feel free to push changes. :)

**Comment by CH1NRU5T** [comment]
Done @the-cyberman 

Do check the PR :)

**Comment by the-cyberman** [comment]
I downloaded your forked repo on my local machine, and found that the
calculator UI is kinda broken. On clicking the "SCI-MODE-ON" button, a new
side palatte appears with the trigonometric/logarithmic functions. The
original idea behind the UI wasn't this simply. I wanted the
scientific-mode buttons to replace the numeric buttons, which doesn't seem
to work in the new code in your forked repo.

**Comment by CH1NRU5T** [comment]
Will try to fix that asap @the-cyberman 
Thanks for pointing it out ✌️

**Comment by truptinaukarkar** [comment]
I suggest adding a calculation history panel to the calculator. It could display recent expressions and results, making the app more user-friendly. I’d be happy to help with the HTML/CSS layout and styling for this feature.
